marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/FlowMvcTest.java

```diff
@@ -0,0 +1,35 @@
+package com.marklogic.hub.central.controllers;
+
+import com.marklogic.hub.central.AbstractMvcTest;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+
+import javax.ws.rs.core.MediaType;
+import java.nio.charset.StandardCharsets;
+
+
+public class FlowMvcTest extends AbstractMvcTest {
+
+    private final static String PATH = "/api/flows";
+
+    @Test
+    void runIngestionStep() throws Exception {
+        installReferenceModelProject();
+        setTestUserRoles("hub-central-user","data-hub-operator");
+        loginAsTestUser();
```

Comment:
No need to change, but you can do these 2 lines in 1 line - loginAsTestUserWithRoles.